packages/ui-app/src/InputExtrinsic/options/section.ts

```diff
@@ -2,27 +2,21 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
-import { Extrinsic$Sections } from '@polkadot/extrinsics/types';
-
+import { Section } from '@polkadot/params/types';
+import { Extrinsic$Sections, Extrinsics } from '@polkadot/extrinsics/types';
 import { DropdownOptions } from '../types';
+import { shouldDisplaySection } from '../../util/shouldDisplaySection';
 
 import map from '@polkadot/extrinsics';
 
-export default function createOptions (type: 'private' | 'public'): DropdownOptions {
+export default function createOptions (type: string = 'private' || 'public'): DropdownOptions {
```

Comment:
Not sure I understand this one - it is not a general string, rather one of 2 values. It should rather (if an issue) be defined as an opaque type (as it was) and if an issue in use at a single point, casted.

Reply:
On closer inspection, this is actually completely wrong now. What is defined here now is an optional parameter that (when not supplied) takes the value of ‘public || private’ which evaluated to the former. It is not a type anymore, rather now you can pass in any string value, ie. even ‘foo’ will work and the type checker will be happy.